pkg/arvo/sys/vane/gall.hoon

```diff
@@ -94,7 +95,15 @@
       agent=(each agent vase)
       =beak
       marks=(map duct mark)
+      sky=(map spur path-state)
```

Comment:
It'd be really nice to have this in the bowl.

Reply:
Done! I had to move `+mop` to lull which is certainly debatable (maybe hoon is better even though it's not necessary?) but I figure we can discuss that once we go through the code.